File: workspace/pset6/cash/cash.py
from cs50 import get_float

total = round(get_float("Change owed:") * 100)

if total < 0:
    total = round(get_float("Change owed:") * 100)

quarters = total // 25
left_over = total - quarters * 25

dimes = left_over // 10
left_over = left_over - dimes * 10

nickels = left_over // 5
left_over = left_over - nickels * 5

total_coins = quarters + dimes + nickels + left_over

print(total_coins)

File: workspace/pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float n;
    do
    {
        n = get_float("How much change?");
    }
    while (n < 0);

    int cents = round(n * 100);

    int quarters = cents / 25;
    int leftover = cents - (quarters * 25);

    int dimes = leftover / 10;
    leftover = leftover - (dimes * 10);

    int nickels = leftover / 5;
    leftover = leftover - (nickels * 5);

    int pennies = leftover / 1;
    leftover = leftover - (pennies * 1);

    int total = quarters + dimes + nickels + pennies;
    printf("%i\n", total);
}